MapboxCoreNavigation/RouteController.swift

```diff
@@ -336,8 +336,9 @@ open class RouteController: NSObject {
         }
     }
     
-    private func update(progress: RouteProgress, with location: CLLocation, rawLocation: CLLocation) {
+    private func update(progress: RouteProgress, with location: CLLocation, rawLocation: CLLocation, upcomingRouteAlerts routeAlerts: Array<UpcomingRouteAlert>) {
```

Comment:
Are alerts always present or do we want this to be an optional?

Reply:
Array itself is non-nil. It is also important to update it each time since "upcoming" alerts do not include already reported ones, so most of the time this array is reset to empty.